.teamcity/src/main/kotlin/promotion/PublishGradleDistribution.kt

```diff
@@ -45,10 +45,14 @@ abstract class PublishGradleDistribution(
             gradleWrapper {
                 name = "Promote"
                 tasks = task
-                gradleParams = """-PcommitId=%dep.${RelativeId("Check_Stage_${this@PublishGradleDistribution.triggerName}_Trigger")}.build.vcs.number% $extraParameters "-PgitUserName=$gitUserName" "-PgitUserEmail=$gitUserEmail" "-PbotGradleGitHubToken=%github.bot-gradle.token%" """
+                gradleParams = """-PcommitId=%dep.${RelativeId("Check_Stage_${this@PublishGradleDistribution.triggerName}_Trigger")}.build.vcs.number% $extraParameters "-PgitUserName=$gitUserName" "-PgitUserEmail=$gitUserEmail" """
```

Comment:
This PR is confusing. It currently moves the command line argument to an environment variable, which I think is a good thing. The description says the PR does something else.

Moreover, I don't understand is why the PR is not against `release6x`.

Reply:
I'll merge this and cherry pick to `release`/`release6x`.